### PR TITLE
Angular standalone sample

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR adds a sample for using MSAL Angular v3 with Angular standalone components and updates documentation.

### CODE REVIEW

**Comment by schmkr** [comment]
question: with v3 of msal-angular, shouldn't the public client be explicitly initialised (with `await client.initialize();`). I don't see that in the example added it, but without the explicit initialization we see in the console these errors:

```
BrowserAuthError: native_broker_called_before_initialize: You must call and await the initialize function before attempting to call any other MSAL API when native brokering is enabled
```

**Comment by jo-arroyo** [comment]
> question: with v3 of msal-angular, shouldn't the public client be explicitly initialised (with `await client.initialize();`). 

@schmkr great question. `initialize()` is called by `handleRedirectObservable`, so as long as the `handleRedirectObservable` is correctly subscribed to (as in this sample) or bootstrapped with the `MsalRedirectComponent` (other msal-angular samples), the public client should be initialized and the error you quoted should not appear. Please see the source code for `handleRedirectObservable` [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/8c382b3b677bf9388154ab6c2863d6951905e91a/lib/msal-angular/src/msal.service.ts#L58).

If you are still seeing that error, please try subscribing to `handleRedirectObservable` as shown in [this sample app](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/cae2279e005b626e17f3f2a29e76e1c10199f821/samples/msal-angular-v3-samples/angular-standalone-sample/src/app/app.component.ts#L34) and [open an issue](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/new/choose) if you are still seeing that error.

File: lib/msal-angular/FAQ.md [hunk 1]
```diff
@@ -43,21 +44,25 @@ Please see [here](https://github.com/AzureAD/microsoft-authentication-library-fo
 
 ### What versions of Angular are supported?
 
-MSAL Angular v3 is in alpha and currently supports Angular 15 and 16.
+MSAL Angular v3 is in beta and currently supports Angular 15 and 16.
```
Comment (peterzenz) [inline lib/msal-angular/FAQ.md]: Another update point for the upcoming release.  @hectormmg

File: samples/msal-angular-v3-samples/angular-standalone-sample/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+import { Route, provideRouter, withDisabledInitialNavigation, withEnabledBlockingInitialNavigation } from '@angular/router';
+import { MsalInterceptor, MSAL_INSTANCE, MsalInterceptorConfiguration, MsalGuardConfiguration, MSAL_GUARD_CONFIG, MSAL_INTERCEPTOR_CONFIG, MsalService, MsalGuard, MsalBroadcastService } from '@azure/msal-angular';
+import { IPublicClientApplication, PublicClientApplication, InteractionType, BrowserCacheLocation, LogLevel, BrowserUtils } from '@azure/msal-browser';
+import { AppComponent } from './app/app.component';
+import { HomeComponent } from './app/home/home.component';
+import { ProfileComponent } from './app/profile/profile.component';
+import { FailedComponent } from './app/failed/failed.component';
+import { environment } from './environments/environment';
+
+const isIE = window.navigator.userAgent.indexOf("MSIE ") > -1 || window.navigator.userAgent.indexOf("Trident/") > -1;
```
Comment (peterzenz) [inline samples/msal-angular-v3-samples/angular-standalone-sample/src/main.ts]: Any isIE references should be gone from our code in our MSAL.js v3 release.  We're dropping IE support in v3.

**Review by peterzenz — APPROVED** [review]

File: samples/msal-angular-v3-samples/angular-standalone-sample/src/main.ts (hunk 2 above)
Comment (jo-arroyo) [reply samples/msal-angular-v3-samples/angular-standalone-sample/src/main.ts]: Makes sense. I've removed all references to it. Thanks!

File: lib/msal-angular/FAQ.md (hunk 1 above)
Comment (tnorling) [inline lib/msal-angular/FAQ.md]: Since GA is imminent
```suggestion
MSAL Angular v3 currently supports Angular 15 and 16.
```

**Review by tnorling — APPROVED** [review]

**Pull request merged**